arcrest/server.py

```diff
@@ -772,14 +772,15 @@ def parentLayer(self):
     @property
     def subLayers(self):
         sub_layers = []
-        for layer in self._json_struct['subLayers']:
-            if layer.get('parentLayer'):
-                value = self._get_subfolder("../%s/" % layer['parentLayer']['id'], MapLayer)
-                sub_layers.append(value)
-            if layer.get('parentLayerId'):
-                value = self._get_subfolder("../%s/" % layer['parentLayerId'], MapLayer)
-                sub_layers.append(value)
-        return sub_layers
+        if 'subLayers' in self._json_struct:
+            for layer in self._json_struct['subLayers']:
+                if layer.get('parentLayer'):
+                    value = self._get_subfolder("../%s/" % layer['parentLayer']['id'], MapLayer)
+                    sub_layers.append(value)
+                if layer.get('parentLayerId'):
+                    value = self._get_subfolder("../%s/" % layer['parentLayerId'], MapLayer)
+                    sub_layers.append(value)
+            return sub_layers
```

Comment:
I would rather leave the `return` out of the `if`-block. When there is no `subLayers`, `sub_layers` is an empty sequence and returning it is just fine.